api/conftest.py

```diff
@@ -168,6 +169,21 @@ def feature(project, environment):
     return Feature.objects.create(name="Test Feature1", project=project)
 
 
+@pytest.fixture()
+def change_request(environment, admin_user):
+    return ChangeRequest.objects.create(
+        environment=environment, title="Test CR", user_id=admin_user.id
+    )
+
+
+@pytest.fixture()
+def feature_state(feature, environment, change_request):
```

Comment:
Should this fixture have a more descriptive name? Perhaps `change_request_feature_state` or something?